docs/operators/csvParserOperator.md

```diff
@@ -121,19 +121,19 @@ CSV Parser is both statically and dynamically partitionable.
 This can be achieved in 2 ways
 
 1. Specifying the partitioner and number of partitions in the populateDAG() method
-  ```java
+```java
   CsvParser csvParser = dag.addOperator("csvParser", CsvParser.class);
   StatelessPartitioner<CsvParser> partitioner1 = new StatelessPartitioner<CsvParser>(2);
   dag.setAttribute(csvParser, Context.OperatorContext.PARTITIONER, partitioner1);
-  ```
+```
 
 2. Specifying the partitioner in properties file.
-  ```xml
+```xml
    <property>
      <name>dt.operator.{OperatorName}.attr.PARTITIONER</name>
      <value>com.datatorrent.common.partitioner.StatelessPartitioner:2</value>
    </property>
-  ```
+```
```

Comment:
This looks OK but there is another issue: the 2 numbered points above both are numbered 1; can you fix that too ?